printing/printer.ml

```diff
@@ -995,3 +998,8 @@ let print_and_diff oldp newp =
         pr_open_subgoals ~proof
     in
     Feedback.msg_notice output;;
+
+let pr_typing_flags flags =
+  str "check_guarded: " ++ bool flags.check_guarded
+  ++ str "\ncheck_positive: " ++ bool flags.check_positive
```

Comment:
Use an `fnl()` instead of `\n`.

Reply:
Done. Thanks

Reply:
Additionally, you can encapsulate the code into a box (e.g. a `v 0 (...)` to ensure that the `fnl` will align the next `check` exactly under the first `check` and not under the beginning of the box which encapsulates the call to `pr_typing_flags` (even if they probably coincide as the code is currently).